cloud/scope/powervs_machine.go

```diff
@@ -1074,3 +1075,26 @@ func (m *PowerVSMachineScope) APIServerPort() int32 {
 	}
 	return infrav1beta2.DefaultAPIServerPort
 }
+
+func (m *PowerVSMachineScope) bucketName() string {
```

Comment:
I know its a code duplication, lets add a todo to handle it later, I think it needs some refactoring

Reply:
Done.